### PR TITLE
Update docs for new release

### PR DESCRIPTION
**Pull request by JakeSCahill**

### CODE REVIEW

File: trinity/0.1/references/troubleshooting.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ When you send a transaction, it has a pending status until it's confirmed.
 
 If a transaction is pending for a long time, make sure that the [Auto-promotion setting](../how-to-guides/change-the-advanced-settings.md) is set to **Enabled**.
 
-**Note:** Auto-promotion is availble on mobile devices only when Trinity is in the foreground.
+:::info:
+Auto-promotion is availble on mobile devices only when Trinity is in the foreground.
```
Comment (cvarley100) [inline trinity/0.1/references/troubleshooting.md]:
```suggestion
Auto-promotion is available on mobile devices only when Trinity is in the foreground.
```

File: trinity/0.1/how-to-guides/send-a-transaction.md [hunk 2]
```diff
@@ -28,6 +34,10 @@ To send to an address in Trinity, it must include the 9-tryte checksum. If you g
 
     ![Sending IOTA tokens](../images/send.jpg)
 
+    :::info:
+    If you minimize Trinity while sending a transaction, the sending process pauses until you reopen it.
```
Comment (cvarley100) [inline trinity/0.1/how-to-guides/send-a-transaction.md]:
```suggestion
    If you minimize Trinity Mobile while sending a transaction, the sending process pauses until you reopen it.
```

File: trinity/0.1/how-to-guides/create-deep-link.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Create a deep link
+
+**Deep links are customizable links that automatically fill the transaction fields in Trinity. You can add deep links to your website or ecommerce platform to make IOTA payments quicker and easier for customers.**
+
+When you click a deep link, Trinity is opened and the parameters of the link are used to fill the transaction fields in your primary account.
```
Comment (cvarley100) [inline trinity/0.1/how-to-guides/create-deep-link.md]: When you click a deep link, Trinity is opened and the parameters of the link are used to fill the transaction fields
```suggestion
When you click a deep link, Trinity is opened and the parameters of the link are used to fill the transaction fields.
```
It fills them for all accounts

File: trinity/0.1/how-to-guides/create-deep-link.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+* `amount`
+* `message`
+
+For example:
+
+```
+iota://XNGPUCURQLLJFGXNDCMROGYNWAZP9AFWSVEUAIWIESOSPYDUPWSPSREEBWJPD9ZWZPAJKBHPLG99DJWJCZUHWTQTDD/?amount=1000000&message=shirt
+```
+
+:::info:
```
Comment (cvarley100) [inline trinity/0.1/how-to-guides/create-deep-link.md]: Don't think this info point is needed. Usually users don't create deep links for themselves to use.

File: trinity/0.1/how-to-guides/create-deep-link.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+Accounts that are linked to hardware wallets can't send value transactions with a message.
+
+If you're using a hardware wallet, you use either an amount or a message in deep links, but not both.
+:::
+
+1. Go to **Settings** > **Advanced settings** and enable deep linking
+
+2. Go back to the home page and click **Receive** to generate a new address
+
+3. Create a deep link, using the new address
```
Comment (cvarley100) [inline trinity/0.1/how-to-guides/create-deep-link.md]:
```suggestion
3. Manually create a deep link outside of Trinity, using the new address
```

File: trinity/0.1/how-to-guides/create-deep-link.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+4. Embed the deep link on your website or ecommerce platform
+
+    For example, on a website you may want to create a clickable link in HTML:
+
+    ```html
+    <a>iota://XNGPUCURQLLJFGXNDCMROGYNWAZP9AFWSVEUAIWIESOSPYDUPWSPSREEBWJPD9ZWZPAJKBHPLG99DJWJCZUHWTQTDD/?amount=1000000&message=shirt</a> 
+    ```
+
+    :::info:
+    You can't add spaces to the message in deep links.
```
Comment (cvarley100) [inline trinity/0.1/how-to-guides/create-deep-link.md]:
```suggestion
    You currently can't add spaces to the message in deep links.
```

File: trinity/0.1/how-to-guides/create-deep-link.md (hunk 4 above)
Comment (JakeSCahill) [reply trinity/0.1/how-to-guides/create-deep-link.md]: Just kept the main point about hardware wallet accounts not being able to send messages and tokens. 

Some users may create deep links with some logic that requires a message (putting the SKU code in the message or something).

**Pull request merged**